src/cpp/cuda.hpp

```diff
@@ -997,8 +997,8 @@ namespace pycuda
       CUstream m_stream;
 
     public:
-      stream(unsigned int flags=0)
-      { CUDAPP_CALL_GUARDED(cuStreamCreate, (&m_stream, flags)); }
```

Comment:
AFAICT, this also needs `cuCtxGetStreamPriorityRange` in order to be meaningfully usable.

Reply:
I implemented it.